CHANGELOG.md

```diff
@@ -14,11 +14,15 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
 #### Major Changes
 
 #### Minor Changes
+- Logginghouse-Client: Add logging-house-client extension 0.1.0
 
 #### Patch Changes
 - Docs: Enhanced starting a Http-Pull over the EDC-Ui documentation
 
 ### Deployment Migration Notes
+- new logging-house-client env for mds:
+  - ```CLEARINGHOUSE_CLIENT_EXTENSION_ENABLED:``` "true"
+  - ```EDC_CLEARINGHOUSE_LOG_URL:``` https://example.com
```

Comment:
- Why are the settings called "...CLEARINGHOUSE.." when it's a Logging-House extension? The term ClearingHouse dates back to the days of the IDS protocol/RAM 4.0 and since the DSP has typically been referred to as a Logging-House , even if there is no DSP-specification for this yet and no DSP-Messages like the old IDS-Protocol "logMessage".
- What is the default value of the Logging-House under which the Logging-House in the MDS can be reached?.

Reply:
1. IDS RAM 4.0 is the current IDS spec. In RAM 5 the Clearinghouse has been referred as Observability. The term Logging-house originates from the MDS. We took the old variables to make the migration easier for customers of a previous MDS EDC 4.x.
2. There is no default logging-house URL. It depends on the environment.